truffle/src/main/ruby/core/rubinius/common/main.rb

```diff
@@ -34,6 +35,22 @@ def include(*mods)
   # Truffle: Do not define #public, #private in Ruby since they need
   # to set the top-level frame visibility when called with no argument.
```

Comment:
I'll have to look this one over again.  At the very least, they should be wrapped in `Truffle.omit` rather than commented out.